autotests/testolmaccount.cpp

```diff
@@ -445,20 +445,18 @@ void TestOlmAccount::claimMultipleKeys()
 void TestOlmAccount::enableEncryption()
 {
     CREATE_CONNECTION(alice, "alice9"_L1, "secret"_L1, "AlicePhone"_L1)
+    alice->syncLoop();
 
+    QSignalSpy createRoomSpy(alice.get(), &Connection::loadedRoomState);
     auto job = alice->createRoom(Connection::PublishRoom, {}, {}, {}, {});
-    QSignalSpy createRoomSpy(job, &BaseJob::success);
     QVERIFY(createRoomSpy.wait(10000));
-    alice->sync();
-    connect(alice.get(), &Connection::syncDone, this, [alice](){
-        alice->sync();
-    });
+
     while(alice->roomsCount(JoinState::Join) == 0) {
```

Comment:
This loop is actually unnecessary because `loadedRoomState()` emission already guarantees at least one room in the room list.